pages/mesosphere/dcos/1.14/administering-clusters/replacing-a-master-node/index.md

```diff
@@ -29,6 +29,8 @@ You can replace a master node in an existing DC/OS cluster. You should keep in m
 
     For more information about backing up the DC/OS identity and access management CockroachDB database, see [How do I backup the IAM database?](/mesosphere/dcos/1.14/installing/installation-faq/#iam-backup)
 
+1. Backup /var/lib/dcos/exhibitor-tls-artifacts if it exists.
```

Comment:
We could provide a command to do that. :)

Reply:
Also, in this context "back up" is a verb phrase and should be two words, not one.